src/styleguide/_data/data.json

```diff
@@ -32,14 +32,31 @@
     "city": "Visakhapatnam",
     "country": "India"
   },
-  "locations": [
-    {}, {}, {}
-  ],
+  "office": {
```

Comment:
wouldn't be better to reuse the `locations` and `location` key that are used in the careers page?

Reply:
Refactored that now